src/app/app.scss

```diff
@@ -1 +1,11 @@
-@import 'assets/nhsuk-8.1.1.min.css'
+@import 'assets/nhsuk-8.1.1.min.css';
+
+html {
+  font-size: 100%;
+  background-color: #fff;
```

Comment:
same here, I don't think we need to specify these colours and font size?

Reply:
taken this off too